lib/chef-browser/helpers.rb

```diff
@@ -115,12 +124,16 @@ def resource_list(resource, data_bag = nil)
 
     def pretty_metadata(key, value)
       case key
-      when 'long_description' then GitHub::Markup.render('README.md', value)
+      when 'long_description' then GitHub::Markup.render_s(GitHub::Markups::MARKUP_MARKDOWN, value)
```

Comment:
The old behavior worked ok, but it looked a bit confusing because the `render` method chose rendering type based on an arbitrary file name passed as the first argument. We're not rendering a file - just a very long string - thus I switched to `render_s` that feels more direct.